src/aap_eda/api/serializers/activation.py

```diff
@@ -562,6 +585,11 @@ class PostActivationSerializer(serializers.ModelSerializer):
         allow_null=True,
         child=serializers.IntegerField(),
     )
+    k8s_service_name = serializers.CharField(
+        required=False,
+        allow_null=True,
+        validators=[validators.check_if_rcf_1035_compatible],
```

Comment:
@hsong-rh this function might be better called check_if_rfc_1035_compliant. In the current name rfc is mistyped as rcf.